### ISSUE TITLE
Python 3.5 docker is using 3.6

### ISSUE THREAD

**Issue by ahartikainen** [arviz-devs/arviz#540]
See https://travis-ci.org/arviz-devs/arviz/jobs/480107251

```
0.01s$ source ~/virtualenv/python3.5/bin/activate
$ python --version
Python 3.5.6
$ pip --version
pip 18.1 from /home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/pip (python 3.5)
before_install.1
11.09s$ docker pull conda/miniconda3
Using default tag: latest
latest: Pulling from conda/miniconda3
Status: Downloaded newer image for conda/miniconda3:latest
before_install.2
0.08s$ docker --version
Docker version 18.06.0-ce, build 0ffa825
install.1
312.12s$ . ./scripts/container.sh --build
Building Docker Image
Sending build context to Docker daemon  8.862MB
Step 1/23 : FROM conda/miniconda3
 ---> 7bd5d4e6b751

...

Step 18/23 : RUN ./scripts/create_testenv.sh
 ---> Running in a266d7389b3d
+ command -v conda
+ PYTHON_VERSION=3.6
+ PYSTAN_VERSION=latest
+ PYRO_VERSION=latest
+ [[ '' != *--global* ]]
+ ENVNAME=testenv_3.6_PYSTAN_latest_PYRO_latest

...

721.37s$ if [[ $NAME == UNIT ]]; then docker run  --mount type=bind,source="$(pwd)",target=/opt/arviz --name arviz_test arviz:latest /bin/bash -c \ "source /root/.bashrc && NUMBA_DISABLE_JIT=1 pytest -v arviz/tests/ --cov=arviz/"; fi
============================= test session starts ==============================
platform linux -- Python 3.6.2, pytest-4.1.1, py-1.7.0, pluggy-0.8.1 -- /usr/local/envs/testenv_3.6_PYSTAN_latest_PYRO_latest/bin/python
cachedir: .pytest_cache
rootdir: /opt/arviz/arviz/tests, inifile: pytest.ini
plugins: cov-2.6.1
collecting ... collected 749 items
```

**Comment by ahartikainen** [comment]
I made a PR https://github.com/arviz-devs/arviz/pull/542

**Comment by ahartikainen** [comment]
Closed by #542
